src/components/navigation/index.tsx

```diff
@@ -21,7 +21,7 @@ export function Navigation() {
     <div
       className={clsx("nav", {
```

Comment:
Better to use `cn`. @lluchkaa [should mentioned it before](https://youtu.be/re2JFITR7TI?si=oQW-CQnXM7-BmXfk).